tests/workflow_management.test.ts

```diff
@@ -272,7 +272,9 @@ describe('workflow-management-tests', () => {
     expect(result.rows[0].attempts).toBe(String(1));
     expect(result.rows[0].status).toBe(StatusString.SUCCESS);
 
-    await reattemptWorkflow(config, null, handle.getWorkflowUUID(), true); // Restart the workflow
+    // Restart the workflow
+    const wfh = await dbosExec.executeWorkflowUUID(handle.getWorkflowUUID(), true);
```

Comment:
But if we change this so the restart path isn't called, is the test really testing anything?